lib/ecto/adapters/clickhouse/connection.ex

```diff
@@ -345,9 +363,9 @@ defmodule Ecto.Adapters.ClickHouse.Connection do
   defp join_qual(:inner), do: " INNER JOIN "
   defp join_qual(t) when t in [:inner_lateral, :array], do: " ARRAY JOIN "
   defp join_qual(t) when t in [:left_lateral, :left_array], do: " LEFT ARRAY JOIN "
-  defp join_qual(:left), do: " LEFT OUTER JOIN "
-  defp join_qual(:right), do: " RIGHT OUTER JOIN "
-  defp join_qual(:full), do: " FULL OUTER JOIN "
+  defp join_qual(:left), do: " LEFT JOIN "
+  defp join_qual(:right), do: " RIGHT JOIN "
+  defp join_qual(:full), do: " FULL JOIN "
```

Comment:
This is not a breaking change as OUTER is assumed for left/right/full joins by default.